docker/openproblems-r-extras/r_requirements.txt

```diff
@@ -14,3 +14,4 @@ shiny@1.4.0.2
 tidyverse@1.3.0
 tidymodels@0.1.2
 reticulate@1.18
+almaan/RCTD
```

Comment:
this is your fork right? are the changes gonna end up in main repo eventually?

Reply:
yes it's mine; I've made a PR in the main; but not sure how fast that will get added. Once, if Dylan approves PR we can change this later on was my idea.